deliverables/README.md

```diff
@@ -31,38 +36,96 @@ What is the role of Javascript in a web page?
+
 What is SCM?  
+SMC stands for Software configuration management, which handles the task of tracking and controlling changes in the software,it is part of the larger cross-discipline field of configuration management. SMC's can also determine what was changed and who changed it in the software.   
+
 What is a VCS?  
+a VCS is a Version Control System which is a system that records changes to a file or set of files over time so that you can recall specific versions later on in the programing process. It allows us to change files back to a previous state, return the entire project back to a previous state, compare changes over time, see who last modified something that might be causing a problem, and lastly who introduced an issue into the program potentially.
+
 Why is Git useful for a developer?  
+Git is usful for developers because it is an easier way for them to submit commands adn write code online. It is the main place where we can put in our work. It is a place to put and develope projects that any developer may have.
+
 Why is Git useful for a team of developers?  
+It is useful for a team of developers because each team member can centralize they work in one place. The creation of the project can happen with many team members and they can all bring they work together and when it is time to finish the project all the cading adn commands are in one place.
+
 How do you create a new Git repository for a project locally?  
+The git inital command creates a new Git repository. We go into our terminal adn then open up sublime text and there we have already copied over the file to there we can start to make our changes to our project. 
+
 How do you create it on Github?  
+You will, once you have changed all the things on a text editor you will through your terminal, use git add and git push to send that project to the Github.
+
 How do you commit changes?  
+To commit your changes simply type git commit on the terminal and the git you just added will commit to the Github.
+
 What is the difference between staging and committing changes?  
+Stage to Commit would be for a file that already has changes being tracked which you've since altered. You would generally only "add" a file once to begin tracking changes, and thereafter you would simply stage the file for commit that you've changed.
+
 What is the difference between committing your changes and pushing them to Github?  
+A commit is a revision so to speak or an individual change to a file or set of files, tt's like when you save a file. Where pushing your changes will actually send all your finished work to a repository such as GitHub.com.
+
 What is the command to check the status of your current repo in git?  
+The command to check our status of our repo is git status
```

Comment:
You can use backticks (`` ` ``) to signify content that's a command, or a variable, or code of any sort.  It formats `like this`